packages/cli/src/services/util.ts

```diff
@@ -77,8 +77,16 @@ export function pathToPosix (relPath: string): string {
 }
 
 export function splitConfigFilePath (configFile?: string): { configDirectory: string, configFilenames?: string[] } {
```

Comment:
How do we handle paths with `~` for the homedirectory? For example `~/my-mac-project/checkly.config.ts`?

I think that this isn't supported by Node.js ([here](https://github.com/nodejs/node/issues/684)) so we might need to pull in a library. Maybe we should pull in [untildify](https://www.npmjs.com/package/untildify) as a dependency.

Reply:
Nevermind, I tested it out. `~` is already handled by the shell.